apps/Unciv/description

```diff
@@ -0,0 +1,3 @@
+An open-source, mod-friendly Android+Desktop remake of Civ V
+
+To run: Menu -> Games -> Sublime Text
```

Comment:
Could you fix the "To run" entry?

Reply:
Fixed in https://github.com/Botspot/pi-apps/pull/1274/commits/277c85b4483299b52082f17881fa6c5667e03748.